src/components/SettingsDialog/SettingsDialog.tsx

```diff
@@ -17,8 +17,8 @@ export const SettingsDialog: FC = () => {
   const {t} = useTranslation();
   const navigate = useNavigate();
   const boardId = useAppSelector((applicationState) => applicationState.board.data!.id);
-  const me = useAppSelector((applicationState) => applicationState.participants?.self.user);
-  const isBoardModerator = useAppSelector((state) => state.participants?.self.role === "MODERATOR" || state.participants?.self.role === "OWNER");
+  const me = useAppSelector((applicationState) => applicationState.participants?.self?.user)!;
```

Comment:
right now, the settings dialog is only available in the board view, where it can be asserted that participants are defined, so this cannot cause problems. however you're correct that this will not be defined everywhere, but that's a problem for #4265. I will soon also add a PR which will attempt to minimize the amount of optional store variables, so we don't need to check or assert for non-null values everywhere.

Reply:
amazing, thanks!